src/sdkToEventsApiConverter.ts

```diff
@@ -339,7 +339,7 @@ export function convertProducts(
             price: sdkProduct.Price,
             quantity: sdkProduct.Quantity,
             coupon_code: sdkProduct.CouponCode,
-            custom_attributes: sdkProduct.Attributes,
+            custom_attributes: sdkProduct.Attributes as Record<string, string>,
```

Comment:
see if this should be string, unknown

Reply:
`Record<string, string>` doesn't work because `EventsApi.Product` requires `custom_attributes` to be `[key: string]: string`